source/_patterns/00-base/_init.global.scss

```diff
@@ -48,17 +48,15 @@ body {
 	line-height: 1.5;
 }
 
+* {
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
+}
+
```

Comment:
that's already part of DB UI Bases `scss/_init.scss` file, do we really need it here? Or do you want to move these declarations later on to DB UI Base anyhow? Than let's leave a _TODO_ comment here.

Reply:
removed it